integration-test/src/java/com/twitter/heron/integration_test/topology/serialization/resources/SerializationTopology.json

```diff
@@ -0,0 +1 @@
+[10]
```

Comment:
I'm not that familiar with how this test framework works, but if we emit the following from the spouts:
```
new CustomObject("A", 10)
new CustomObject("B", 20)
```
and then do an increment bolt, why would we expect to get just [10]?